isset needs lowercase

gtz42 opened the pull request:
while the getter and setter apear to ignore the (sub)variables case, the isset actulle only works with lowercase.  Also correct a concationated test.

gtz42 committed:
isset needs lowercase

while the getter and setter apear to ignore the (sub)variables case, the isset actulle only works with lowercase.  Also correct a concationated test.

alex1702 reviewed (changes requested):
change "showRightSidebar" to "showrightsidebar" in exampleSite/config.toml
 and README.md for 0.17 compatibility

gtz42 committed:
fix is simpler

By just testing if the parameter is true, the code can be simplified.  If the key does not exist, it's NIL and therefore false.

alex1702 commented:
Oh Ok nice
